<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>516</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Auto Excel</string>
  </property>
  <widget class="QListWidget" name="list_of_executions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="file_path_entry">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>testing</string>
   </property>
  </widget>
  <widget class="QLabel" name="directory_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Save Path:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="file_name_entry">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>30</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>test.xlsx</string>
   </property>
  </widget>
  <widget class="QLabel" name="file_name_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>File Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="path_name_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Value will Be Variable</string>
   </property>
  </widget>
  <widget class="QLabel" name="data_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prefix_entry">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="suffix_entry">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="min_spinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="max_spinBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QLabel" name="prefix_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Prefix:</string>
   </property>
  </widget>
  <widget class="QLabel" name="range_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Range:</string>
   </property>
  </widget>
  <widget class="QLabel" name="suffix_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Suffix:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_selection_button">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>310</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Selection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Input</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_input_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Input</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_info_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="generate_excel_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Excel File</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
